src/services/helpers/service.js

```diff
@@ -31,7 +31,7 @@ module.exports = function setup(app) {
 			const notificationPlatform = app.get('NOTIFICATION_PLATFORM');
 
 			if (!notificationPlatform) {
-				throw new MethodNotAllowed('Required Env NOTIFICATION_PLATFORM is not defined');
+				logger.warning('Required Env NOTIFICATION_PLATFORM is not defined');
```

Comment:
Ich glaube, wenn die Variable wirklich benötigt ("required") wird, dann reicht es hier nicht zu loggen, sondern der Fehler ist schon korrekt. Hat meiner Meinung nach auch nichts mit dem PR zu tun.

Reply:
der test sollte beim anwendungsstart ausgeführt werden

Reply:
Wenn im development und tests keine notification Services existiert, soll nichts verschickt werden. Sondern es soll bis zum logging gehen. Daher ist sie hier nicht required. 
Ich würde auch in Frage stellen ob das überhaupt beim verwenden geprüft werden sollte. 
Meiner Meinung nach sollte das beim initialisieren des Services passieren.

Reply:
@tofixx kann ich machen, aber nur als warning. Sonst startet der server nicht mehr wenn kein notification Service hinterlegt ist.

Reply:
dann müssen wir die Stelle etwas tiefgreifender umbauen...
NOTIFICATION_PLATFORM sollte ins default.schema.json als optionaler Parameter, wenn der gesetzt ist soll alles funktionieren, wenn der nciht gesetzt ist sollte es ein Verhalten geben das für Dev Sinnvoll ist, am besten die Notifications stattdessen dem logging übergeben.

Die Frage ist ob das als Teil des PRs sinnvoll ist oder in ein eigenes ticket geschoben werden sollte... und ich hatte bisher noch keine Probleme mit dem Fehler obwohl ich keine notification_plattform habe...